Gemfile.lock

```diff
@@ -497,7 +498,8 @@ GEM
     zeitwerk (2.6.12)
 
 PLATFORMS
-  ruby
+  x86_64-darwin-21
+  x86_64-linux
```

Comment:
Not sure if it's a good idea to whitelisting platforms...  This might have unexpected consequences in production